cmd/snap/cmd_snapshot.go

```diff
@@ -0,0 +1,303 @@
+package main
```

Comment:
Stopped the review here. Done only _client/_ for now. The following is next.

Reply:
Thank you!